### PR TITLE
NOISSUE - Fix disconnect things

### PR DESCRIPTION
**Pull request by 0x6f736f646f**
Signed-off-by: 0x6f736f646f <blackd0t@protonmail.com>

### What does this do?
It fixes the disconnect thing function

### Which issue(s) does this PR fix/relate to?
It doesn't relate to previous issues

### List any changes that modify/break current functionality
There was no disconnect things function as indicated by the docs [here](https://docs.mainflux.io/api/#disconnect)

### Have you included tests for your changes?
Yes.

### Did you document any new/modified functionality?
No

### CODE REVIEW

**Review by drasko — APPROVED** [review]
LGTM

File: lib/things.py [hunk 1]
```diff
@@ -119,3 +119,16 @@ def disconnect(self, channel_ids, thing_ids, token):
...
+        payload = {
+          "thing_ids": thing_ids,
+          "channel_ids": channel_ids,
+        }
+        http_resp = requests.put(self.url + "/disconnect/", headers={"Authorization": token}, json=payload)
+        mf_resp = response.Response()
+        if http_resp.status_code != 204:
+            mf_resp.error.status = 1
+            mf_resp.error.message = errors.handle_error(errors.things["delete"], http_resp.status_code)
+        return mf_resp
```
Comment (fbugarski) [inline lib/things.py:134]: Add new line at the end of the file.

**Review by fbugarski — CHANGES_REQUESTED** [review]

File: lib/things.py (hunk 1 above)
Comment (0x6f736f646f) [reply lib/things.py]: Added the new line

**Review by fbugarski — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**